assets/javascripts/discourse/initializers/salesforce-topic-list-class.js.es6

```diff
@@ -0,0 +1,22 @@
+import discourseComputed from "discourse-common/utils/decorators";
+import { withPluginApi } from "discourse/lib/plugin-api";
+
+function initializeWithApi(api) {
+  api.modifyClass("component:topic-list-item", {
+    @discourseComputed()
+    unboundClassNames() {
```

Comment:
Instead of overriding core, perhaps this is a good time to introduce an interface in core?

Reply:
I can do it in a separate PR. This PR is already a big one. And I want to merge it quickly :)